apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -10,6 +13,46 @@ const ProjectWidget = React.createClass({
     projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
   },
 
+  statics: {
+    setupProjectWidget() {
+      $.ajax({
+        method: 'GET',
+        url: `/v3/channels`,
+        dataType: 'json'
+      }).done(projectLists => {
+        const widget = document.getElementById('projects-widget');
+        projectLists.sort((a, b) => {
+          if (a.updatedAt < b.updatedAt) {
+            return 1;
+          } else {
+            return -1;
+          }
+        });
+        projectLists = projectLists.filter(project => {
+          return !project.hidden;
+        });
+        projectLists = projectLists.slice(0,4);
+        let convertedProjectList = [];
+        for (let i = 0; i < projectLists.length; i++){
+          let convertedProject = {
+            name: projectLists[i].name,
+            channel: projectLists[i].id,
+            thumbnailUrl: projectLists[i].thumbnailUrl,
+            type: projectLists[i].projectType,
+            updatedAt: projectLists[i].updatedAt
+          };
+          convertedProjectList.push(convertedProject);
+        }
+        ReactDOM.render(
+          <Provider store={getStore()}>
+            <ProjectWidget projectList={convertedProjectList}/>
+          </Provider>,
+          widget
```

Comment:
This feels wrong. We have one component calling a static method on another component, which is then doing a render itself? I'm not sure if setupProjectWidget should actually be a method on ProjectWidget